index.js

```diff
@@ -30,6 +30,7 @@ exports.clean = async function () {
 };
 
 exports.deploy = async function (options) {
+    await eventtracking.askForOptInAndAccountID(options);
```

Comment:
this is not necessarily called by user, actually more likely to be used in some deploy script on CI, etc